src/main/java/com/github/vkorobkov/jfixtures/config/structure/util/TableMatcher.java

```diff
@@ -0,0 +1,34 @@
+package com.github.vkorobkov.jfixtures.config.structure.util;
+
+
+import com.github.vkorobkov.jfixtures.util.CollectionUtil;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+@FunctionalInterface
+public interface TableMatcher {
+
+    default boolean tableMatches(String table) {
+        return flatTablesToMatch().anyMatch(pattern -> {
+            if (pattern.startsWith("/")) {
+                if (Pattern.compile(pattern.substring(1)).matcher(table).matches()) {
+                    return true;
```

Comment:
could be simplified?
just return Pattern.compile(pattern.substring(1)).matcher(table).matches();